src/remoteCollection/collect.js

```diff
@@ -41,7 +42,7 @@ function sendRemoteRequest(generator) {
 
     // If token is present, add to request header.
     if (generator.token) {
-      const accessToken = generator.token.accessToken;
+      const accessToken = generator.token.accessToken || generator.token.access_token;
```

Comment:
is this a valid expectation? didn't find that in the docs or tests

Reply:
Its universal standard either accessToken or access_token will be returned from remote source.